events/nickname_events/wc_nickname_events.txt

```diff
@@ -0,0 +1,141 @@
+###################
+# Nickname Events
+###################
+
+namespace = WCNIC
+
+### the Old Murkeye
+WCNIC.0001 = { # Nickname for when the character rule long
+    type = character_event
+	title = WCNIC.0001.t
+
+	desc = {
+		triggered_desc = {
+			trigger = { has_culture_group = culture_group:murloc_group }
+			desc = WCNIC.0001.desc.m
+		}
+		triggered_desc = {
+			trigger = { has_culture_group = culture_group:human_group }
```

Comment:
Only human cultures? Why?

Reply:
What cultures could have the event too? I was thinking of other titles for other cultures. Like "the Elder" for orcs.

Reply:
> What cultures could have the event too? I was thinking of other titles for other cultures. Like "the Elder" for orcs.

Cultures with the same life span as humans. Definitely not elves because their old begins in age 2000.
I don't think the Old doesn't fit other cultures like orcs.